qds_sdk/cloud/aws_cloud.py

```diff
@@ -0,0 +1,172 @@
+        self.set_compute_config(use_account_compute_creds, compute_access_key, compute_secret_key, role_instance_profile)
+        self.set_location(aws_region, aws_availability_zone)
+        self.set_network_config(bastion_node_public_dns, persistent_security_groups, master_elastic_ip, vpc_id, subnet_id)
+
+
+    def set_compute_config(self,
+                           use_account_compute_creds=None,
+                           compute_access_key=None,
+                           compute_secret_key=None,
+                           role_instance_profile=None):
+        self.compute_config['use_account_compute_creds'] = use_account_compute_creds
+        self.compute_config['compute_access_key'] = compute_access_key
+        self.compute_config['compute_secret_key'] = compute_secret_key
+        self.compute_config['role_instance_profile'] = role_instance_profile
+
+
+    def set_location(self,
+                     aws_region=None,
+                     aws_availability_zone=None):
+        self.location['aws_region'] = aws_region
+        self.location['aws_availability_zone'] = aws_availability_zone
+
+    def set_network_config(self,
+                           bastion_node_public_dns=None,
+                           persistent_security_groups=None,
+                           master_elastic_ip=None,
+                           vpc_id=None,
+                           subnet_id=None):
+        self.network_config['bastion_node_public_dns'] = bastion_node_public_dns
+        self.network_config['persistent_security_groups'] = persistent_security_groups
+        self.network_config['master_elastic_ip'] = master_elastic_ip
+        self.network_config['vpc_id'] = vpc_id
+        self.network_config['subnet_id'] = subnet_id
+
+
+
+
+
+    def set_cloud_config_from_arguments(self, arguments):
+        self.set_cloud_config(compute_access_key=arguments.compute_access_key,
+                              compute_secret_key=arguments.compute_secret_key,
+                              use_account_compute_creds=arguments.use_account_compute_creds,
+                              aws_region=arguments.aws_region,
+                              aws_availability_zone=arguments.aws_availability_zone,
+                              role_instance_profile=arguments.role_instance_profile,
+                              vpc_id=arguments.vpc_id,
+                              subnet_id=arguments.subnet_id,
+                              persistent_security_groups=arguments.persistent_security_groups,
+                              bastion_node_public_dns=arguments.bastion_node_public_dns,
+                              master_elastic_ip=arguments.master_elastic_ip)
+
+    def create_parser(self, argparser):
+
+        # compute settings parser
+        compute_config = argparser.add_argument_group("compute config settings")
+        compute_creds = compute_config.add_mutually_exclusive_group()
+        compute_creds.add_argument("--enable_account_compute_creds",
```

Comment:
Please replace underscore (`_`) with dash (`-`) in `enable_account_compute_creds `.